### PR TITLE
Model development

### PR DESCRIPTION
**Pull request by DRASHHTIPATEL**
Uploaded denoising autoencoder model code. Looking forward to your comments and review.

### CODE REVIEW

**Review by s1ashd0t — APPROVED** [review]
Approved. just one comment: I thought we were using Keras instead of Torch

**Review by lukebushur — COMMENTED** [review]
As long as we can support both Pytorch and Keras, this looks good! Is it expected for the training and validation loss to be so high?

**Review by ayushdh96 — APPROVED** [review]
Looks good to me. Project's coming alive. Kudos!
